test/unit/mocks/MockST_ETH.sol

```diff
@@ -3,40 +3,70 @@ pragma solidity ^0.8.24;
 
 // https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/StETH.sol
 import {ERC20} from "src/Common.sol";
+import {IStETH} from "src/interface/external/lido/IStETH.sol";
 
-contract MockSTETH is ERC20 {
-    uint256 private _pooledEthPerShare = 1e18; // Start with 1:1 ratio
-    mapping(address => uint256) public shares;
+contract MockSTETH is IStETH, ERC20 {
```

Comment:
fixed this mock so it behaves like steth.

before this, the stETH value was not 1:1 with ETH

This has rebasing working, and can accrue rewards through the function addRewards(). The function does not change the stETH:ETH ratio but increases everyone's stETH balance.